system/applications/modules/virt-manager/default.nix

```diff
@@ -19,4 +20,17 @@ mkIf (cfg.virtualisation.virtManager) {
   users.users = mapAttrs (user: _: {
     extraGroups = [ "libvirtd" ];
   }) cfg.users;
+
+  boot.kernelParams = [
+    # Allows passthrough of independent devices, that are members of larger IOMMU groups
+    # It only affects kernels with ACS Override support. Ex: CachyOS, Liquorix, Zen
+    "pcie_acs_override=downstream,multifunction"
+  ] ++ (
+      # Enable PCIe passthrough
+      if (cpus.amd.enable)
```

Comment:
use lib.optional here.

Reply:
like this?
```nix
++ (optional cpus.amd.enable "amd_iommu=on"
++ (optional cpus.intel "intel_iommu=on"
```

Reply:
Yep, but, without "(", after "++" though.